### ISSUE TITLE
Remove typescipt 'paths' as it's not compatible with nodejs require()

### ISSUE THREAD

**Issue by burtonator** [hicommonwealth/commonwealth#7218]
## Describe the bug

I want to remove this as it's going to continually cause us problems

```
The paths configuration in tsconfig.json is indeed not directly compatible with Node.js require(). 
The paths configuration is used by TypeScript's compiler (tsc) to resolve module paths during 
the TypeScript compilation process. It helps TypeScript resolve module imports to the correct 
file paths.

However, Node.js itself doesn't understand or interpret tsconfig.json. When Node.js encounters
 require() statements, it doesn't automatically resolve them based on TypeScript's paths 
configuration. Instead, it relies on its own module resolution algorithm, which typically 
involves searching for modules in node_modules directories or using relative paths.
```

What's happening now is that tsconfig is configured with 'paths' which is supported by webpack.

tsc won't complain, nor will webpack, so we don't get any type of static failure here.

What happens is node fails at runtime and we won't be able to catch the bug until it's impacted in prod

There are 94 places in the code where this needs to be fixed.

### Initial conditions

Flaky as the build could fail.

## Reproduction steps

- require from a 'path' and it will fail.

**Comment by burtonator** [comment]
@kurtisassad @timolegros this should address the 'utils' import issue that I found in the RSS PR.  

Do you have any comments here?  

This way we can kill this bug once and for all.

**Comment by burtonator** [comment]
@Rotorsoft This is another issue I found that we should try to cleanup.

**Comment by burtonator** [comment]
Actually this is actually in 2000 places :-/. Maybe a better solution is to fix this via eslint-diff and require a proper package import path.

**Comment by burtonator** [comment]
we can probably fix this by using a proper package reference in yarn to "@commonwealth/core" rather than doing a paths entry which isn't going to be reliable.

**Comment by timolegros** [comment]
Yea so this is related to our usage of tsc-alias and is the long term goal after #6796 (review this for context) but as you have found out there are quite a few places where paths is used which is why we haven't addressed this directly yet. Ideally we could address this issue for the server (which has fewer issues to correct) but leave it in the client for the time being because Webpack handles the path derivation.

Using eslint diff might be a good idea here to phase these imports out over time on the client.

**Comment by burtonator** [comment]
@timolegros Thanks! This really helps. Part of what I'm trying to do is understand some of the tech debt and tribal knowledge so we can move forward over these things.  I'll take a look.  Maybe there is some low hanging fruit or ways to move forward without breaking anything.

**Comment by burtonator** [comment]
OK.  I think the solution here is to probably use yarn workspaces and have everything use modules.  

Basically we would have the depend on the packages directly like

```json
    "@hicommonwealth/core": "0.0.0",
    "@hicommonwealth/chains": "0.0.0",
    "@hicommonwealth/adapters": "0.0.0",
    "@hicommonwealth/model": "0.0.0"
```

and we would build those via the standard yarn reactor for workspaces.

This would mean we can properly isolate our packages, have a tsconfig for each, with hopefully strict mode and a tight typescript config, and without importing files outside of the project.

I checked I think the only reason this wouldn't work is because we're importing files outside of the root of the project.  

If we fix that I think we can switch quickly.

**Comment by Rotorsoft** [comment]
I think we have made good progress on the package isolation front, but we might still have some remaining issues... wondering where you see those imports pointing to external files outside of the root?
